ddtrace/contrib/trace_utils.py

```diff
@@ -243,6 +252,17 @@ def set_http_meta(
     request_headers=None,  # type: Optional[Mapping[str, str]]
     response_headers=None,  # type: Optional[Mapping[str, str]]
     retries_remain=None,  # type: Optional[Union[int, str]]
+    tracer=None,
+    raw_uri=None,
+    query_object=None,
+    format_query_object=_identity,
```

Comment:
For now we can inline the formatting functions since they are the same across integrations.

When we get to the point where another formatter is required we can revisit the design